### PR TITLE
Fix gofmt and govet

### PR DESCRIPTION
**Pull request by wupeka**
Fixed some issues with formatting/vetting, minor changes mostly.

### CODE REVIEW

File: packaging/commands/interface.go [hunk 1]
```diff
@@ -99,7 +99,6 @@ func NewPackageCommander(series string) (PackageCommander, error) {
 	default:
 		return NewAptPackageCommander(), nil
 	}
-	return nil, nil
```
Comment (perrito666) [inline packaging/commands/interface.go]: I am pretty sure this return is needed, I am not sure all versions of the compiler are that smart, what version did you try this with?

File: packaging/config/interface.go [hunk 2]
```diff
@@ -49,8 +49,6 @@ func NewPackagingConfigurer(series string) (PackagingConfigurer, error) {
 	default:
 		return NewAptPackagingConfigurer(series), nil
 	}
-
-	return nil, nil
```
Comment (perrito666) [inline packaging/config/interface.go]: same as with previous return

File: packaging/commands/interface.go (hunk 1 above)
Comment (wupeka) [inline packaging/commands/interface.go]: 1.6.3

Comment (wupeka) [inline packaging/commands/interface.go]: I've verified and even 1.2.1 thinks it's unreachable.

**Comment by wupeka** [comment]
$$merge$$

**Pull request merged**